packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/NftCollectionImageSmall.tsx

```diff
@@ -37,9 +35,7 @@ const NftCollectionImageSmall: React.FC<Props> = (props) => {
     <Wrapper>
       {isVerified ? (
         <IconWrapper>
-          <Icon label='verified' color='orange400'>
-            <IconVerified />
-          </Icon>
+          <IconVerified label='verified' color={PaletteColors['orange-400']} />
```

Comment:
I know this is about the colors but the above destructuring is hurting my eyes, and them not having defaults and being treated with `||` below makes it a bit sadder